tests/core/pyspec/eth2spec/test/electra/block_processing/test_process_consolidation_request.py

```diff
@@ -857,7 +758,7 @@ def test_incorrect_no_source_execution_withdrawal_credential(spec, state):
 )
 @spec_test
 @single_phase
-def test_incorrect_no_target_execution_withdrawal_credential(spec, state):
+def test_incorrect_target_bls_credential(spec, state):
```

Comment:
For consistency with `test_incorrect_target_with_eth1_credential`, this should be:
```suggestion
def test_incorrect_target_with_bls_credential(spec, state):
```